\doxysection{Vinarstvi.\+exceptions.\+Record\+Not\+Found\+Exception Class Reference}
\hypertarget{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception}{}\label{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception}\index{Vinarstvi.exceptions.RecordNotFoundException@{Vinarstvi.exceptions.RecordNotFoundException}}


Represents an exception that is thrown when a record is not found.  


Inheritance diagram for Vinarstvi.\+exceptions.\+Record\+Not\+Found\+Exception\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=2.000000cm]{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception}
\end{center}
\end{figure}
\doxysubsubsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
\mbox{\hyperlink{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a89a1f35583fb31652d0b03ca02f4cba8}{Record\+Not\+Found\+Exception}} ()
\begin{DoxyCompactList}\small\item\em Initializes a new instance of the Record\+Not\+Found\+Exception class. \end{DoxyCompactList}\item 
\mbox{\hyperlink{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a12741c37e22d685a6b4750ff17a8928c}{Record\+Not\+Found\+Exception}} (string message)
\begin{DoxyCompactList}\small\item\em Initializes a new instance of the Record\+Not\+Found\+Exception class with the specified message. \end{DoxyCompactList}\item 
\mbox{\hyperlink{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a4d12566c9b61297300602ead18d9fce9}{Record\+Not\+Found\+Exception}} (string message, Exception inner\+Exception)
\begin{DoxyCompactList}\small\item\em Initializes a new instance of the Record\+Not\+Found\+Exception class with a specified error message and a reference to the inner exception that is the cause of this exception. \end{DoxyCompactList}\end{DoxyCompactItemize}


\doxysubsection{Detailed Description}
Represents an exception that is thrown when a record is not found. 

\doxysubsection{Constructor \& Destructor Documentation}
\Hypertarget{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a89a1f35583fb31652d0b03ca02f4cba8}\label{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a89a1f35583fb31652d0b03ca02f4cba8} 
\index{Vinarstvi.exceptions.RecordNotFoundException@{Vinarstvi.exceptions.RecordNotFoundException}!RecordNotFoundException@{RecordNotFoundException}}
\index{RecordNotFoundException@{RecordNotFoundException}!Vinarstvi.exceptions.RecordNotFoundException@{Vinarstvi.exceptions.RecordNotFoundException}}
\doxysubsubsection{\texorpdfstring{RecordNotFoundException()}{RecordNotFoundException()}\hspace{0.1cm}{\footnotesize\ttfamily [1/3]}}
{\footnotesize\ttfamily Vinarstvi.\+exceptions.\+Record\+Not\+Found\+Exception.\+Record\+Not\+Found\+Exception (\begin{DoxyParamCaption}{ }\end{DoxyParamCaption})}



Initializes a new instance of the Record\+Not\+Found\+Exception class. 

\Hypertarget{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a12741c37e22d685a6b4750ff17a8928c}\label{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a12741c37e22d685a6b4750ff17a8928c} 
\index{Vinarstvi.exceptions.RecordNotFoundException@{Vinarstvi.exceptions.RecordNotFoundException}!RecordNotFoundException@{RecordNotFoundException}}
\index{RecordNotFoundException@{RecordNotFoundException}!Vinarstvi.exceptions.RecordNotFoundException@{Vinarstvi.exceptions.RecordNotFoundException}}
\doxysubsubsection{\texorpdfstring{RecordNotFoundException()}{RecordNotFoundException()}\hspace{0.1cm}{\footnotesize\ttfamily [2/3]}}
{\footnotesize\ttfamily Vinarstvi.\+exceptions.\+Record\+Not\+Found\+Exception.\+Record\+Not\+Found\+Exception (\begin{DoxyParamCaption}\item[{string}]{message }\end{DoxyParamCaption})}



Initializes a new instance of the Record\+Not\+Found\+Exception class with the specified message. 


\begin{DoxyParams}{Parameters}
{\em message} & The error message that explains the reason for the exception.\\
\hline
\end{DoxyParams}
\Hypertarget{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a4d12566c9b61297300602ead18d9fce9}\label{class_vinarstvi_1_1exceptions_1_1_record_not_found_exception_a4d12566c9b61297300602ead18d9fce9} 
\index{Vinarstvi.exceptions.RecordNotFoundException@{Vinarstvi.exceptions.RecordNotFoundException}!RecordNotFoundException@{RecordNotFoundException}}
\index{RecordNotFoundException@{RecordNotFoundException}!Vinarstvi.exceptions.RecordNotFoundException@{Vinarstvi.exceptions.RecordNotFoundException}}
\doxysubsubsection{\texorpdfstring{RecordNotFoundException()}{RecordNotFoundException()}\hspace{0.1cm}{\footnotesize\ttfamily [3/3]}}
{\footnotesize\ttfamily Vinarstvi.\+exceptions.\+Record\+Not\+Found\+Exception.\+Record\+Not\+Found\+Exception (\begin{DoxyParamCaption}\item[{string}]{message,  }\item[{Exception}]{inner\+Exception }\end{DoxyParamCaption})}



Initializes a new instance of the Record\+Not\+Found\+Exception class with a specified error message and a reference to the inner exception that is the cause of this exception. 


\begin{DoxyParams}{Parameters}
{\em message} & The error message that explains the reason for the exception.\\
\hline
{\em inner\+Exception} & The exception that is the cause of the current exception.\\
\hline
\end{DoxyParams}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
\mbox{\hyperlink{_record_not_found_exception_8cs}{Record\+Not\+Found\+Exception.\+cs}}\end{DoxyCompactItemize}
